modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+[id="migration-migrating-applications-on-prem-to-cloud_{context}"]
+= Migrating an application from on-premises to a cloud-based cluster
+
+You can migrate from a source cluster that is behind a firewall to a cloud-based destination cluster by establishing a network tunnel between the two clusters. The `crane tunnel-api` command establishes such a tunnel by creating a VPN tunnel on the source cluster and then connecting to a VPN server running on the destination cluster. The VPN server is exposed to the client using a load balancer address on the destination cluster.
+
+A service created on the destination cluster exposes the source cluster's API to {mtc-short}, which is running on the destination cluster.
+
+.Prerequisites
+
+* The system that creates the VPN tunnel must have access and be logged in to both clusters.
+* It must be possible to create a load balancer on the destination cluster. Refer to your cloud provider to ensure this is possible.
+* Have names prepared to assign to namespaces, on both the source cluster and the destination cluster, in which to run the VPN tunnel. These namespaces should not be created in advance. For information on namespace rules, see \https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names.
+* When connecting multiple firewall-protected source clusters to the cloud cluster, each source cluster requires its own namespace.
+* OpenVPN server is installed on the destination cluster.
+* OpenVPN client is installed on the source cluster.
+* When configuring the source cluster in {mtc-short}, the API URL takes the form of \https://proxied-cluster.${namespace}.svc.cluster.local:8443.
+** If you use the API, see _Create a MigCluster CR manifest for each remote cluster_.
+** If you use the {mtc-short} web console, see _Migrating your applications using the {mtc-short} web console_.
+* The {mtc-short} web console and Migration Controller must be installed on the target cluster.
+
+.Procedure
+
+. Install the `crane` utility:
++
+[source,terminal,subs="+quotes"]
+----
+$ podman cp $(podman create registry.redhat.io/rhmtc/openshift-migration-controller-rhel8:v1.7.0):/crane ./
+----
+
+. Log in remotely to a node on the source cluster and a node on the destination cluster.
+
+. Obtain the cluster context for both clusters after logging in:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc config view
+----
+
+. Establish a tunnel by entering the following command on the command system:
++
+[source,terminal,sub="+quotes"]
+----
+$ crane tunnel-api [--namespace _<namespace>_] \
+      --destination-context _<destination-cluster>_ \
+      --source-context _<source-cluster>_
```

Comment:
```suggestion
$ crane tunnel-api [--namespace <namespace>] \
      --destination-context <destination-cluster>\
      --source-context <source-cluster>
```

Currently, the OpenShift docs repo differs from the SSG and ISG's guidance for user-defined variables. We just use brackets without italics. See the [Quick markup reference](https://github.com/openshift/openshift-docs/blob/9a12c390cfe11f474e7234b0601b74fa320e6654/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) in the repo guidelines.